configure.ac

```diff
@@ -317,6 +317,20 @@ AM_CONDITIONAL([MAINTAINER_TOOLS],[test x"${i_do_have_maintainer_tools}" = xyes]
 AC_SUBST([MAINTAINER_TOOLS],["${i_do_have_maintainer_tools}"])
 
 #--------------------------------------------------------------------------
+# Check for autotrace and potrace (don't do anything with this yet)
```

Comment:
What's the point of adding this check? None of the autotrace code is dependent on a compile time check to see if autotrace is present or not. autotrace/potrace could be installed/removed after fontforge was compiled, and this would make no difference to fontforge - as long as it's in PATH or the AUTOTRACE variable is set when the autotracing function is used.